SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -801,4 +807,109 @@ void FLocalDeploymentManager::SetAutoDeploy(bool bInAutoDeploy)
 	bAutoDeploy = bInAutoDeploy;
 }
 
+void SPATIALGDKSERVICES_API FLocalDeploymentManager::TakeSnapshot(UWorld* World, bool bUseStandard,
+																  FSpatialSnapshotTakenFunc OnSnapshotTaken)
+{
+	FHttpModule& HttpModule = FModuleManager::LoadModuleChecked<FHttpModule>("HTTP");
+	TSharedRef<IHttpRequest> HttpRequest = HttpModule.Get().CreateRequest();
+
+	HttpRequest->OnProcessRequestComplete().BindLambda(
+		[World, bUseStandard, OnSnapshotTaken](FHttpRequestPtr HttpRequest, FHttpResponsePtr HttpResponse, bool bSucceeded) {
+			if (!bSucceeded)
+			{
+				UE_LOG(LogSpatialDeploymentManager, Error, TEXT("Failed to trigger snapshot at '%s'; received '%s'"),
+					   *HttpRequest->GetURL(), *HttpResponse->GetContentAsString());
+				if (OnSnapshotTaken != nullptr)
+				{
+					OnSnapshotTaken(false, FString());
```

Comment:
For undeclared vars used in methods I'd like it if we can include the var name in /* InVarName */ style for maintainability